### ISSUE TITLE
Turn off cloud services but still record to SD card?

### ISSUE THREAD

**Issue by radical6** [ghoost82/mijia-720p-hack#20]
Is it possible to disable cloud services but still record video to the SD card?

**Comment by fliphess** [comment]
Not really. The recording feature is part of the cloud suite so it's either both or none.

**Comment by thosuperman** [comment]
Would anyone know how to make Xiaomi Mijia 720p record video to google cloud?
